packages/frontend/vite.config.ts

```diff
@@ -49,13 +50,14 @@ export default defineConfig({
                     vega: ['vega', 'vega-lite'],
                     ace: ['ace-builds', 'react-ace/lib'],
                     modules: [
-                        'pegjs',
+                        // TODO: removed because of PNPM
```

Comment:
don't we need these modules ?

Reply:
these are for bundling packages together. not a blocker but ideally I should figure out why they do not work anymore